src/kili/services/label_data_parsing/json_response.py

```diff
@@ -66,23 +113,75 @@ def __init__(
                 job_payload=job_response,
             )
 
-    def __getitem__(self, job_name: str) -> "job_response_module.JobPayload":
-        """Returns the JobPayload object corresponding to the job name."""
-        if job_name not in self._json_data:
-            raise JobNotExistingError(job_name)
-        return self._json_data[job_name]
-
-    @property
-    def jobs(self) -> "ParsedJobs":
-        """Returns self."""
-        return self
-
-    def to_dict(self) -> Dict:
+    def to_dict(self) -> Dict[str, Dict]:
         """Returns the parsed json response as a dict."""
         ret = {job_name: job_payload.to_dict() for job_name, job_payload in self._json_data.items()}
         ret = {k: v for k, v in ret.items() if v}  # remove empty json responses
         return ret
 
+
+def _is_video_job(project_info: Project, json_response: Dict) -> bool:
+    """Returns True if the json response is a video job, False otherwise."""
+    if "VIDEO" not in project_info["inputType"]:
+        return False
+
+    if not all(frame_id.isdigit() for frame_id in json_response.keys()):
+        return False
+
+    for frame_response in json_response.values():
+        if len(frame_response) == 0:
+            continue
+
+        for job_name in frame_response.keys():
+            if not (isinstance(job_name, str) and job_name in project_info["jsonInterface"]):
```

Comment:
why not to return a JobNotExistingError(job_name) here as in other jobs ? ?

Reply:
this method is just a helper to determine if it's a video json response or not